### ISSUE TITLE
Fetch custom field guide instructions not matching

### ISSUE THREAD

**Issue by geraldleungontario** [frappe/erpnext#41175]
### Information about bug

I'm trying to follow this guide:
https://frappeframework.com/docs/user/en/guides/app-development/fetch-custom-field-value-from-master-to-all-related-transactions

However, step #2 is not possible. In my "Fetch From" field, I get a dropdown selector (screenshot below) and can only choose from the list of Doc Type. In that documentation example, would that mean you would have to create a Doc Type for GSTIN, and define GSTIN in Supplier as a Link to the GSTIN Doc Type, then you can choose GSTIN from the "Fetch From" field in step 3?

![Fetch From](https://github.com/frappe/erpnext/assets/114938861/2aec3726-444b-486d-87de-c4b8ebc42cdf)

### Module

stock, buying

### Version

frappe/erpnext:v15.20.3

### Installation method

docker

**Comment by Nihantra-Patel** [comment]
Since you are searching at the purchase order item level and the supplier field is at the purchase order level, there's a limitation. If you want to access the GSTIN at the purchase order level, it is possible. However, if you wish to include the GSTIN at the purchase order Item level, you must add the supplier field. This is necessary because the GSTIN is stored in the supplier master. Without the supplier field, retrieving the GSTIN is not feasible. I hope this clarifies your query.


If you have a doubt, then raise the question in the Forum: https://discuss.frappe.io

**Comment by geraldleungontario** [comment]
I did not follow the guide instructions exactly in my previous example. I made the mistake of using a Purchase Order Item instead of Purchase Order. The issue still happens when trying to customize a Purchase Order. I am
 unable to enter "supplier.gstin" like in the guide instructions and only get to choose from the drop down list and "supplier.gstin" is not available. Should the guide instructions be updated?

![Fetch From 1](https://github.com/frappe/erpnext/assets/114938861/9789b8e5-bb41-45e6-97a0-fa58ea198f62)
![Fetch From](https://github.com/frappe/erpnext/assets/114938861/f808508d-3beb-42da-9ee1-0587ab27ed7b)

I found a working video guide from
[https://discuss.frappe.io/t/fetch-from-doc-type/119898/4](url)
